sickbeard/notifiers/trakt.py

```diff
@@ -227,13 +227,23 @@ def test_notify(self, username, password, disable_ssl):
         api: The api string to use
         username: The username to use
         password: The password to use
+        blacklist_name: slug of trakt list used to hide not interested show
 
         Returns: True if the request succeeded, False otherwise
         """
         try:
             trakt_api = TraktAPI(sickbeard.TRAKT_API_KEY, username, password, disable_ssl, sickbeard.TRAKT_TIMEOUT)
             trakt_api.validateAccount()
-            return "Test notice sent successfully to Trakt"
+            if blacklist_name and blacklist_name is not None:
+                trakt_lists = trakt_api.traktRequest("users/" + username + "/lists")
+                found = False
+                for trakt_list in trakt_lists:
+                    if (trakt_list['ids']['slug'] == blacklist_name):
+                        return "a Test notice sent successfully to Trakt"
```

Comment:
Sorry, i forgot to change back it.

Giovanni
Il 25/mar/2015 19:59 "Alexandre Beloin" <notifications@github.com> ha
scritto:

> In sickbeard/notifiers/trakt.py
> <https://github.com/SiCKRAGETV/SickRage/pull/1637#discussion_r27154354>:
>
> >
> >          Returns: True if the request succeeded, False otherwise
> >          """
> >          try:
> >              trakt_api = TraktAPI(sickbeard.TRAKT_API_KEY, username, password, disable_ssl, sickbeard.TRAKT_TIMEOUT)
> >              trakt_api.validateAccount()
> > -            return "Test notice sent successfully to Trakt"
> > +            if blacklist_name and blacklist_name is not None:
> > +                trakt_lists = trakt_api.traktRequest("users/" + username + "/lists")
> > +                found = False
> > +                for trakt_list in trakt_lists:
> > +                    if (trakt_list['ids']['slug'] == blacklist_name):
> > +                        return "a Test notice sent successfully to Trakt"
>
> @gborri <https://github.com/gborri> Can you change this, because a and b
> aren't very user friendly description?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SiCKRAGETV/SickRage/pull/1637/files#r27154354>.
>